### ISSUE TITLE
Could you, please, share usd scene with Ford Mustang?

### ISSUE THREAD

**Issue by jswark** [pablode/gatling#6]

**Comment by pablode** [comment]
Hi @jswark,

I'm unsure whether I'm allowed to redistribute the model since I got it from https://wirewheelsclub.com/.

So here's how I modified it: I opened it in Blender with [AMD's Hydra plugin](https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon) and manually assigned materials. The MaterialX project ships some example materials that can be used, for instance chrome, glass and car paint. The colors can be obtained from the example renders. Alternatively, there are some manually authored color values here: https://github.com/knightcrawler25/GLSL-PathTracer

Maybe my just open-sourced [glTF->USD+MaterialX converter](https://github.com/pablode/guc) can help you too?
